### DISCUSSION TITLE
How to report/fix documentation issues?

### DISCUSSION THREAD

**Discussion by ohemelaar** [outline/outline#4335] [Ideas]
I was reading the hosting docs when I found a [broken link](https://app.getoutline.com/doc/authentication-7ViKRmRY5o) on the page about [local development](https://app.getoutline.com/s/770a97da-13e5-401e-9f8a-37949c19f97e/doc/local-development-5hEhFRXow7).

Where should issues in documentation be reported? Is it possible to suggest fixes in a PR?

Maybe this could be stated in the README, under Contributing:
> - Developer happiness and documentation. See *link to how to contribute to docs*

**Comment by tommoor** [comment]
This actually seems to be a bug in the way shared document links are generated on custom domains, thanks for reporting
